<?xml version="1.0"?>
<launch>
	<node name="image_view" pkg="image_view" type="image_view" ns="agri_mapper">
		<remap from="image" to="tree_detector/image" />
		<param name="window_name" value="Image" />
	</node>
	<node name="circles_image_view" pkg="image_view" type="image_view" ns="agri_mapper">
		<remap from="image" to="tree_detector/detection_image" />
		<param name="window_name" value="Detected" />
	</node>
</launch>
